models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import Column, Integer, String, func, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "author"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(127), unique=True, nullable=False)
+    bio = Column(String(511), nullable=True)
+
+
+class DBBook(Base):
+    __tablename__ = "book"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), nullable=False)
+    summary = Column(String(1023), nullable=True)
+    publication_date = Column(DateTime(timezone=True), server_default=func.now())
```

Comment:
if this field means the time when book was published - then it would be pretty strange that book is published at the same time it was added to your system:)
And imagine situation: in the future, you'll want to add some logic to `bulk_add` which basically will add many books to your system at once. It would be weird if an author will have 10-15 books with the same publication date.